meta-mender-core/recipes-mender/mender-configure/mender-configure.inc

```diff
@@ -0,0 +1,17 @@
+DESCRIPTION = "Mender add-on for device configuration."
+HOMEPAGE = "https://mender.io"
+
+RDEPENDS_${PN} = "dbus curl mender-client (>= 2.5)"
+
+FILES_${PN} = " \
+    ${datadir}/mender/modules/v3/mender-configure \
+    ${datadir}/mender/inventory/mender-inventory-mender-configure \
+"
+
+S = "${WORKDIR}/git"
+
+do_install() {
```

Comment:
Propose to also add:

```
do_install_append_mender-image() {
    install -d -m 755 ${D}/data/mender-configure
    ln -s /data/mender-configure ${D}/var/lib/mender-configure
}
```

Reply:
Right. I'll add it and modify the tests accordingly.